bsconfig.json

```diff
@@ -29,5 +29,5 @@
   "warnings": {
     "error": "+5"
   },
-  "ppx-flags": ["graphql_ppx/ppx"]
+  "ppx-flags": []
```

Comment:
I think we can just remove this entry entirely rather than keeping it around as an empty array.

```suggestion
```